docs/paylinks_and_payment_request.md

```diff
@@ -0,0 +1,87 @@
+    4. If the user selects to trigger this payment handler, trigger [PaymentRequest.show()](https://w3c.github.io/payment-request/#show-method).
+    5. The user agent would then drop the response from PaymentRequest.show(). (Payment links are designed for merchants that have back-end integration with payment apps, where the back-end (server-to-server) calls are responsible for communication of a successful transaction.)
+3. Else if the `<scheme>` is not "https", then see [How to handle non-HTTPS schemes?](#how-to-handle-non-https-schemes) below!
+
+## Open Questions
+
+### When would the UX be shown to the user to offer them the payment handler?
+
+Firstly, it is likely that we would specify (normatively or non-normatively)
+that the browser wait until document load has settled before triggering any UX
+for payment links on the page. We would likely also specify a short pause
+*after* a payment link is newly added to the page, to allow for collecting
+multiple payment links together (see [Handling multiple paylinks on the
+page](#handling-multiple-paylinks-on-the-page)).
+
+In addition, it is important for user agents to make sure that payment links do
+not become a vector for spamming users with un-wanted browser UX. This is also
+a problem for Payment Request, and is why the spec [allows browsers to require
+a transient user activation](https://w3c.github.io/payment-request/#show-method)
+to call `show()`, and generally allows the user agent to reject `show()` calls
+if it believes they are not legitimate.
+
+### Handling multiple paylinks on the page
+
+A site may want to include multiple payment links on the same page (perhaps
+coming from integrations with different PSPs or payment method libraries). This
+can likely be treated similarly to a single PaymentRequest call with multiple
+[PaymentMethodData](https://w3c.github.io/payment-request/#dom-paymentmethoddata)),
+if we specify some logic around 'collecting together' multiple payment links on
+the page.
+
+### Is it useful for pages to be able to have an event listener for the outcome, or a way to hook into the Payment Request that is triggered?
+
+This is definitely feasible (e.g. via defining `onpaymentrequestshow` or
+`onpaymentrequestcomplete`, etc), and simply needs us to determine if its useful.
+
+### How to handle non-HTTPS schemes?
+
+With the current facilitated payment links proposal, [Chromium
+intends](https://github.com/WICG/paymentlink/pull/15) to support the following
+non-HTTPS schemes: `duitnow`, `shopeepay`, and `tngd`. Some of these are
+"single entity" schemes which could instead be represented by a `https` URL
+(e.g., https://shopeepay.com/), while others are more like protocols that could
+be served by multiple independent payment apps.
+
+Handling payment method 'protocols', rather than specific organizations, is a
+problem that Payment Request has discussed historically but never fully solved.
+
```

Comment:
I think you could say more here, namely that we have the concept of standardized payment methods in the Payment Method specification, and we've experimented with them, but we have (1) deprecated basic-card and (2) not heard demand for new standardized payment methods (outside of 'spc'). However, it might be that in a regulated ecosystem, standardized payment methods could be more "acceptable."

Reply:
Done, PTAL.